docs/server/concepts/data_store.mdx

```diff
@@ -18,7 +18,13 @@ This is what we built the `DataAdapter`/`DataStore` for.
 DataAdapters allow you to plug in your own storage solution as a cache for the Statsig server sdk.
 
 They define a simple API that you have to implement: `initialize`, `get`, `set`, and `shutdown` for setup, reading, writing, and closing.
-When a Statsig SDK is initialized with a `DataAdapter`, it will rely on that first as the source of truth for the project definition, before reaching out to Statsig servers. Whenever it gets an up to date version of the project from our api, it will cache it locally using the DataAdapter.
+
+When a Statsig SDK is initialized with a `DataAdapter`, it will rely on that first as the source of truth for fetching the project configuration (includes all experiments, gates and their targeting/allocation rules), before reaching out to Statsig servers. Whenever it gets an up to date version of the project from our api, it will cache it locally using the DataAdapter.
```

Comment:
I know you didn't write this, but i'd re-write it to be more clear:

When the Statsig SDK is initialized with a `DataAdapter`, it will rely on the `DataAdapter` as the source of truth for your project config definitions (includes all experiments, configs, gates and their targeting/allocation rules). Post-initialization, the SDK will continue to poll Statsig servers for updates and whenever it gets an up to date version of the project from our api, it will save it to the DataAdapter, unless you explicitly configure the DataAdapter to be used as the source of truth for polling (`supportsPollingUpdatesFor`)

Reply:
There's actually a bit of repeated stuff with what you added below. So many just consolidating the 2 would be better

Reply:
@kenny-statsig I consolidated and added some high level "use cases"!